src/icon-manifest/supportedExtensions.ts

```diff
@@ -226,6 +226,7 @@ export const extensions: IFileCollection = {
     { icon: 'favicon', extensions: ['favicon.ico'], filename: true, format: FileFormat.svg },
     { icon: 'fbx', extensions: ['fbx'], format: FileFormat.svg },
     { icon: 'firebase', extensions: ['.firebaserc'], filename: true, format: FileFormat.svg },
+    { icon: 'firebase_firestore', extensions: ['firestore.rules'], filename: true, format: FileFormat.svg },
```

Comment:
Better use simply `firestore` as the `icon` attribute value.

Reply:
Changed :)

Reply:
Not that it needs to change again, but I'll just say there is no Firestore on its own without Firebase, which is why I included `firebase_` in the initial `icon` attribute value.
I also prefer the more succinct `firestore`, but just wanted to make the distinction that Firestore is a sort of child of Firebase, if that affects any existing patterns `vscode-icons` may be already using.

Reply:
Fair enough.